locustfile.py

```diff
@@ -0,0 +1,118 @@
+import random
+import json
+import pdb
+
+from locust import HttpLocust, task, TaskSet, TaskSequence
+from faker import Faker
+
+FILE = 'tmp/runtime/development/be/application/mounted/platform/credentials.json'
+
+# def config():
+#   return json.loads(open(FILE).read())
+
+# def access_key_id(self):
+#   return self.config()[2]['credential']['access_key_id']
+
+# def secret_access_key(self):
+#   return self.config()[2]['secret']  
+
+def authorization():
+  return "Basic AGKQQTPKMZQYQJIMIAMM:Li3hJQOXrwtxp_ytIXM8jv2w7d_OX62CBQ4O9d-SOTSKUO3Egw6MfA"
+
+# def create_iam_user(self):
+#   payload =  { "data": { "type": "users", "attributes": { "username": Faker().name(), "time_zone": "SGT" } } }
+#   self.client.post('/iam/users', data=json.dumps(payload), headers={"authorization": authorization(), 'content-type': 'application/vnd.api+json'})  
+
+def create_cognito_user(self, identifier):
+  payload =  { "data": { "type": "users", "attributes": { "primary_identifier": identifier } } }
+  return self.client.post('/cognito/users', data=json.dumps(payload), headers={"authorization": authorization(), 'content-type': 'application/vnd.api+json'} )  
+
+def login_cognito_user(self, identifier):
+  payload = { "data": { "attributes": { "primary_identifier": identifier } } }
+  return self.client.post('/cognito/login', data=json.dumps(payload), headers={"authorization": authorization(), 'content-type': 'application/vnd.api+json' } )
+
```

Comment:
What happened to the IAM stuff that was originally in here? We still need that. These are testing different things